diff --git a/index.js b/index.js
index 79f587973d8ad5d1f108e86282468a6f38a0d8c1..68073f04baa41dc01ba3878c7de69c3789880d38 100644
--- a/index.js
+++ b/index.js
@@ -1,7 +1,7 @@
 const { parse } = require('scss-parser')
 const parse2 = require('gonzales-pe').parse
 const { readFileSync } = require('fs')
-const globAll = require('glob-all')
+const { glob } = require('glob')
 
 const shouldParse = ['rule', 'selector', 'block']
 const shouldKeep = ['id', 'class', 'attribute']
@@ -72,7 +72,11 @@ function sanitizeArgs(arr) {
   if (arr.some(s => typeof s !== 'string')) throw `Oops! Something passed wasn't a string.`
 
   // Ensure absolute paths for filenames, especially if globs were passed.
-  arr = globAll.sync(arr, { absolute: true })
+  const files = []
+  for (const pattern of arr) {
+    files.push(...glob.sync(pattern, { absolute: true }))
+  }
+  arr = files
 
   // If, at the end of it all, we have nothing, leave empty-handed.
   if (!arr.length) {
