diff --git a/dist/checkers/eslint/main.js b/dist/checkers/eslint/main.js
index 27a775515049aa60f23c2778632b86d2db6d3513..1a8b6d88eb72e16395e83057b8e1dc0da8552234 100644
--- a/dist/checkers/eslint/main.js
+++ b/dist/checkers/eslint/main.js
@@ -60,19 +60,9 @@ const createDiagnostic = (pluginConfig) => {
       };
       let eslint;
       if (pluginConfig.eslint.useFlatConfig) {
-        const {
-          FlatESLint,
-          shouldUseFlatConfig
-        } = require2("eslint/use-at-your-own-risk");
-        if (shouldUseFlatConfig == null ? void 0 : shouldUseFlatConfig()) {
-          eslint = new FlatESLint({
-            cwd: root
-          });
-        } else {
-          throw Error(
-            "Please upgrade your eslint to latest version to use `useFlatConfig` option."
-          );
-        }
+        eslint = new ESLint({
+          cwd: root
+        });
       } else {
         eslint = new ESLint(eslintOptions);
       }
